datatypes/src/raster/operations/blit.rs

```diff
@@ -37,23 +36,13 @@ impl<T: Pixel> Blit<RasterTile2D<T>> for RasterTile2D<T> {
             .tile_geo_transform()
             .coordinate_to_grid_idx_2d(source.spatial_bounds().upper_left());
 
-        let source_bounds = source.grid_array.bounding_box();
-        let shifted_bounds = GridBoundingBox::new_unchecked(
-            source_bounds.min_index() + offset,
-            source_bounds.max_index() + offset,
-        );
-
-        let shifted_source = Grid {
-            shape: shifted_bounds,
-            no_data_value: source.grid_array.no_data_value,
-            data: source.grid_array.data,
-        };
+        let shifted_source = source.grid_array.shift_by_offset(offset);
 
         self.grid_array.grid_blit_from(shifted_source)?;
         Ok(())
     }
 }
-
+/*/
```

Comment:
Why did you remove it kind of but not completely? Don't we need the blit operation?

Reply:
i wanted to discuss this. we do not use the code anywhere and it will require us to first materialize each tile since blitting into is only possible on materialized tiles. I can add this or remove the code.